orangewidget/gui.py

```diff
@@ -230,16 +231,37 @@ def miscellanea(control, box, parent,
     :param sizePolicy: the size policy for the box or the control
     :type sizePolicy: QSizePolicy
     """
+    if addSpace is not None:
+        warnings.warn("'addSpace' has been deprecated. Use gui.separator instead.",
+                      DeprecationWarning, stacklevel=3)
     for prop, val in kwargs.items():
-        if prop == "sizePolicy":
-            control.setSizePolicy(QSizePolicy(*val))
+        if prop.startswith('box_'):
+            prop = prop[4:]
+            target = box
+        elif prop.startswith('parent_'):
+            prop = prop[7:]
+            target = parent
         else:
-            getattr(control, "set" + prop[0].upper() + prop[1:])(val)
+            target = control
+
+        dashsplit = prop.split('_')
```

Comment:
No. If you are not using this at least 100x in Orange widgets, no. If we are, we need a better solution.

Reply:
I've removed the unnecessary magic.

Reply:
Thanks, now even I can understand it. :)